bitnami/cilium/README.md

```diff
@@ -0,0 +1,602 @@
+
+The command deploys Cilium on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Configuration and installation details
+
+### [Rolling VS Immutable tags](https://docs.vmware.com/en/VMware-Tanzu-Application-Catalog/services/tutorials/GUID-understand-rolling-tags-containers-index.html)
+
+It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
+
+Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+### External KeyValue Store support
+
+You may want to have Cilium connect to an external KeyValue Store rather than installing one inside your cluster. Typical reasons for this are to use a managed service, or to share a common store for all your applications. To achieve this, the chart allows you to specify credentials for an external database with the [`externalKvstore` parameter](#parameters). You should also disable the etcd installation with the `etcd.enabled` option. Here is an example:
+
+```console
+etcd.enabled=false
+externalKvstorehost.enabled=true
+externalKvstorehost.endpoints[0]=external-kvstore-host-0:2379
+externalKvstorehost.endpoints[1]=external-kvstore-host-1:2379
+```
+
+### TLS secrets
+
+TLS support can be enabled in the chart by setting the `tls.enabled` parameter to `true`.
+
+It is necessary to create a secret containing the TLS certificates and pass it to the chart via the `tls.existingCASecret` and `tls.hubble.existingSecret` parameters. Both secrets should contain a `tls.crt` and `tls.key` keys including the certificate and key files respectively. For example: create the CA secret with the certificates files:
+
+```console
+kubectl create secret generic ca-tls-secret --from-file=./tls.crt --from-file=./tls.key
+```
+
+You can manually create the required TLS certificates or relying on the chart auto-generation capabilities. The chart supports two different ways to auto-generate the required certificates:
+
+- Using Helm capabilities. Enable this feature by setting `tls.autoGenerated.enabled` to `true` and `tls.autoGenerated.engine` to `helm`.
+- Relying on CertManager (please note it's required to have CertManager installed in your K8s cluster). Enable this feature by setting `tls.autoGenerated.enabled` to `true` and `tls.autoGenerated.engine` to `cert-manager`. Please note it's supported to use an existing Issuer/ClusterIssuer for issuing the TLS certificates by setting the `tls.autoGenerated.certManager.existingIssuer` and `tls.autoGenerated.certManager.existingIssuerKind` parameters.
+
+### %%OTHER_SECTIONS%%
```

Comment:
Remove this

Reply:
done at https://github.com/bitnami/charts/pull/26374/commits/129afd439cbffb07203c84e7db2880f82caa370f